docs/helpers/Eventline.md

```diff
@@ -0,0 +1,40 @@
+# Eventline
+
+If Eventline is provided then the `value` property is mandatory.
+
+When using Eventline with graph then `x axis type` must be **timeseries** it is mandatory.
+
+-   [Eventline](#eventline)
+    -   [JSON Properties](#json-properties)
+        -   [Required](#required)
+        -   [Optional](#optional)
+    -   [Structure](#structure)
+
+## JSON Properties
+
+### Required
+
+| Property Name | Expected         | Description                                 |
+| ------------- | ---------------- | ------------------------------------------- |
+| value         | string (ISO8601) | Position where eventline needs to be placed |
+| color         | string (ISO8601) | color of the eventline                      |
+
+### Optional
+
+| Property Name | Expected | Default   | Description                                       |
+| ------------- | -------- | --------- | ------------------------------------------------- |
+| style         | object   | undefined | Any strokeDashArray value to add dash to the line |
```

Comment:
Feel like we would want a default here. If the consumer doesnt pass anything then how does the eventline show up? Solid line?

Reply:
Yes it will be a solid line if the user does not provide any value.

Reply:
Shouldnt an `eventline` always be dotted?

Reply:
For our use case, it will be dotted but there might be scenario consumers want to use it as a solid line.

Reply:
If they want a solid line they might as well use a dateline instead.

Reply:
ok, I will make the 2,2 as the default value for the eventline.